packages/pn-personagiuridica-webapp/src/api/external-registries/external-registries-routes.ts

```diff
@@ -13,9 +13,10 @@ const API_EXTERNAL_REGISTRIES_BASE_PATH = `${PA_SEGMENT}/${API_VERSION_SEGMENT}`
 const API_GET_ALL_ACTIVATED_PARTIES = `${API_EXTERNAL_REGISTRIES_BASE_PATH}/${API_ACTIVATED_PARTIES_SEGMENT}`;
 
 // APIs
-export function GET_ALL_ACTIVATED_PARTIES() {
+export function GET_ALL_ACTIVATED_PARTIES(payload: any) {
```

Comment:
set proper type for payload param and remove "any" usage

Reply:
do it